### DISCUSSION TITLE
Metadata discussion

### DISCUSSION THREAD

**Discussion by nicocopez** [h2020charisma/ramanchada2#90] [Ideas]
Here we can discuss on the metadata we want to include

**Comment by georgievgeorgi** [comment]
- exposure time - intigration times(ms) (Note the typo)
- xaxis units
- safe threshold for saturation - yaxis max
- sample if known and applicable

**Comment by nicocopez** [comment]
I would suggest:
- Integration time
- Polarization (on the collection path, in dependence to the incident polarization. Ex. parallel to excitation polarization, or perpendicular, etc.)

Not very sure about the next ones, but could be interesting discussing it:
- Pulse duration, distance between pulses (for pulsed laser excitation) --> Otherwise, we can specify continuous wavelength or pulsed
-  Sample orientation (difficult to define or to include as a parameter, but it could be useful/necessary for some comparisons)
- Type of objective: air objective / oil inmersion objective /...
-

**Reply by nicocopez** [reply]
- add the extra info about the spectrometer ( Slit size, grating,...?) and the pinhole size (if any, for confocal systems - note that Renishaw systems are "confocal" in just one axis using the spectrometer slit, and the pixel size for the other axis)

**Reply by nicocopez** [reply]
- Question: can we exchange  somehow collection fiber diameter by the fiber numerical aperture?

**Reply by mabanares** [reply]
sd

**Comment by vedina** [comment]
# Experiment setup

- Instrument make and model
- Wavelength (nm)
- Numerical aperture
- Laser power of acquisition, %
- Maximum laser power (mW)
- Humidity during acquisition, %
- Temperature during acquisition, °C
- optical path settings
    -  gratings
    -  collection_optics
    - slit_size
    -  pin_hole_size
    - collection_fibre_diameter

# Sample 
- material id (short name), e.g. PST
- material name (polystyrene)
- supplier 
- info on chemistry (if available , e.g. CAS, SMILES, InChI)

# Metadata on the experiment 
- who have done it (organsation, person, etc)
- when was done
- in what context (e.g. investigation like "polystyrene project" or "round robin 123"

**Comment by nicocopez** [comment]
### **Discussion on the 14-15 / Nov/2023 at the Charisma meeting at VUB in Brussels**

**Enrique's and James' List:**
- Manufacturer
- Model / serial number
- Device type: FT-IR, dispersive, scanning
- Laser wavelength (nm)
- Laser power (mW) or laser power (%) and actual maximum
- Optical path
- Number of data points
- Data calibration : x-axis (type) - Ne, y-axis (type)
- X-units (px#, nm, Raman shift)
- Y-units (a.u., a.u.c. (intensity correction))
- Data pre-processing: Pre-treatment. background correction (file path), offset correction, Cosmic ray filter,.

**Further notes**
- We agree that we need the skeleton or minimum information to reconstruct the spectra. Minimum according to us for all processes. The objective is to have the info enough to run the calibration.
- Integration time, number of accumulations, etc.
- Binding data???? saying whether the spectra has corrupted pixels (hot? dead? pixel).
- (primary) Raw data in pixel number. Manufacturer transforms pixel number in wavenumber or whatever.
- Enrique: Distinguish between: Raw data, primary data, secondary data
- How is it done in Elodiz: A new spectrometer provides pixel number and intensity. The firmware will transform that into nanometers and intensity. Some manufacturer provide the pixel number and other nanometers directly.
- James: There are systems that just provide data in raman shift. So the calibration protocol starts from there. Raw data is in pixels. Enrique: The problem is that some manufacturers sometimes dont tell what is the preprocessing of data.
- James: Encourage manufacturers to give certain info, for example slit opening. 
- Egon: Link things that are needed for the use cases. THat could be useful rather than what is or what is not required, as Dirk suggested.
- Nina ask for a list of manufacturers.

-

**Comment by vedina** [comment]
NeXuS definitions fork https://github.com/h2020charisma/definitions 
The goal is to create definition for Raman spectra, it should go into https://github.com/h2020charisma/definitions/tree/main/contributed_definitions 

NXmpes (multidimensional photoelectron spectroscopy.) might be a good start https://github.com/h2020charisma/definitions/blob/main/contributed_definitions/NXmpes.nxdl.xml . It is [already accepted definition developed by FAIRmat](https://fairmat-nfdi.github.io/nexus-fairmat-proposal/9636feecb79bb32b828b1a9804269573256d7696/classes/contributed_definitions/NXmpes.html#nxmpes)

**Comment by vedina** [comment]
NeXus definition for XPS peak fitting https://github.com/h2020charisma/nexus_definitions_fairmat/blob/fairmat/contributed_definitions/nyaml/NXxpcs.yaml
